packages/jest-worker/src/workers/processChild.ts

```diff
@@ -59,6 +66,12 @@ const messageListener: NodeJS.MessageListener = request => {
 };
 process.on('message', messageListener);
 
+function sendParentMessageHeartbeat() {
+  if (process && process.send) {
```

Comment:
```suggestion
  if (process?.send) {
```

when is `process` not defined?